.travis.yml

```diff
@@ -10,12 +10,6 @@ go:
  - 1.11.x
  - master
 
-env:
```

Comment:
Please revert this. While we can get around what this was introduced to fix with the new cache, in cases where there are gonum-wide changes, this will cause a timeout.

Reply:
not sure about that (the timeout).
I mean: the cache helps to shrink the build time from PR to PR.
within a PR, adding or removing the `cache:` directive won't help a bit.

also: this very PR was a gonum-wide change as far as it was concerned: everything got rebuilt.
besides, a PR times out when at least one of its build-matrix elements takes more than 1h or when there's no printout for more than 10mn (IIRC.)

leveraging the builtin Go build/test cache and running the 4 `go test -tags=XXX` in a row, in the same PR, should make sure we stay within time budget.

Reply:
Divinding the work into tag-based units also helps quickly see what is a cause. If you cannot have this and the cache, then this needs further discussion, but separating the different builds is useful.

Reply:
(moving this discussion to the main PR thread)